invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides

```diff
@@ -58,3 +58,17 @@
     }
   }
 }
+
+
+.ui.items {
+  & > .item.carousel {
```

Comment:
what is the reason to limit the styling only to immediate child component ?

Reply:
It's just overriding how it was defined in SUI